scss/_variables.scss

```diff
@@ -1204,7 +1204,7 @@ $form-check-input-indeterminate-bg-image:       url("data:image/svg+xml,<svg xml
 
 $form-check-input-disabled-opacity:        null !default;
 $form-check-label-disabled-opacity:        null !default;
-$form-check-btn-check-disabled-opacity:    null !default;
+// Boosted mod: no $form-check-btn-check-disabled-opacity
```

Comment:
Can't we keep this Sass var and apply it even if `null` just to ensure compatibility with Bootstrap like it's done for `$form-check-label-disabled-opacity` for instance?